### ISSUE TITLE
Be able to navigate a screen with tab key

### ISSUE THREAD

**Issue by zecakeh** [etesync/etesync-notes#88]
Right now, if we try to navigate the app with the tab key, although it's technically possible, there's no visual feedback most of the time so it's hard to know which element is focused.

This would improve the accessibility of the web app.

**Comment by tasn** [comment]
Yeah, as a keyboard user I'd love it too... The problem is with react-native-paper, I don't know if it's very web friendly.

**Comment by zecakeh** [comment]
I believe that the `react-native-paper` PR mentionned in [#87](https://github.com/etesync/etesync-notes/issues/87#issue-781158368) would mainly solve our problems since, if they use Pressable, we can have the hovered and focused states in react-native-web.

By the way, we should upgrade to react-native-web 0.14.0+ to have the hovered state.

**Comment by tasn** [comment]
Feel free to open a PR, happy to upgrade it!
